source/server/spongineer/incompatible.rst

```diff
@@ -52,6 +52,25 @@ Hammer Core
 
 - Solution: Change ``<entry key="World.ITickable.Override">true</entry>`` to ``false`` in the ``/asm/hammercore.xml`` file
 
+ICBM Classic
+~~~~~~~~~~~~
+- Versions: All
+- Problem: Missiles stop in midair and appear to lag in place.
+- Solution: Add the following mod specific configuration to the ``sponge/global.conf`` entity-activation-range-> mods section
+
+.. code-block:: none
+
+            icbmclassic {
```

Comment:
IMO we should add the wrapping sections (entity-activation-range -> mods) as well, that way it is easier to locate.
Maybe with a comment like `# important part start`.